stable/nginx-ingress/templates/controller-service.yaml

```diff
@@ -0,0 +1,29 @@
+{{- $root := . -}}
+{{- range .Values.controller.services }}
+---
```

Comment:
My specific use-case is to have the same nginx-ingress controllers exposed via two different ELBs (one internet-facing and one VPC-internal). But now that I think about it, that shouldn't be strictly necessary. I'll play with it over the weekend.